projects/ui-policy-studio-angular/src/lib/components/step-form/gio-ps-step-form.component.scss

```diff
@@ -54,6 +54,44 @@ $typography: map.get(gio.$mat-theme, typography);
     overflow: auto;
     flex: 1 1 auto;
     padding: 16px;
+
+    &__markdown {
+      ::ng-deep table {
```

Comment:
It seems to me that this is dangerous. 
What do you think about reversing it? 
Keep the deep but clearly indicate a specific class.

Reply:
Done